[TrilinosApp] Error in tests

happens with 2 cores:
~~~
======================================================================
FAIL: test_trilinos_levelset_convection (test_trilinos_levelset_convection.TestTrilinosLevelSetConvection)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philipp/software/Kratos_MasterBranch/applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py", line 110, in test_trilinos_levelset_convection
    self.assertAlmostEqual(max_distance, 0.7904255118014996)
AssertionError: 0.7333041045431626 != 0.7904255118014996 within 7 places
~~~

I think there have been some changes recently but I don't know which

--- a/applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py
+++ b/applications/TrilinosApplication/tests/test_trilinos_levelset_convection.py
@@ -107,8 +107,8 @@ class TestTrilinosLevelSetConvection(KratosUnittest.TestCase):
         min_distance = comm.MinAll(min_distance)
         max_distance = comm.MaxAll(max_distance)
 
-        self.assertAlmostEqual(max_distance, 0.7904255118014996)
-        self.assertAlmostEqual(min_distance,-0.11710292469993094)
+        self.assertAlmostEqual(max_distance, 0.7333041045431626)
+        self.assertAlmostEqual(min_distance,-0.06371359024393104)
 
 class TestTrilinosLevelSetConvectionInMemory(TestTrilinosLevelSetConvection):
 

